packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,165 @@
+
+Before building a ChatBot, make sure that you have justified it as an appropriate solution by asking yourself these questions:
+- What are the users’ goals?
+- How in-depth is the assistance the user will need?
+- Does human assistance better serve your users?
+- How is a ChatBot superior to online documentation, contextual support or wizards?
+- What data sources or abilities can this ChatBot leverage to assist your users?
+
+Do not create a ChatBot simply for the sake of novelty.
+
+### When to use a ChatBot 
+
+Use a ChatBot to offer your users on-demand help at any time, including:
+- Technical support and troubleshooting.
+- Product information and documentation.
+- Sales and product recommendations.
+- Training and onboarding.
+- System monitoring and alerts.
+- Community engagement .
+- Feedback collection and surveys.
+
+### When not to use a ChatBot
+
+Do not use a ChatBot when: 
+- A task could be accomplished more efficiently through the UI. 
+- A process is very complex or could take a long time.
+- A real human is needed for sensitive or emotional topics.
+
+## Behavior 
+
+### Accessing a ChatBot
+
+Users can enter a conversation with a ChatBot by clicking on the toggle. Once the ChatBot window opens, the toggle will change to display an "angle down" icon to indicate that clicking the toggle again will minimize the ChatBot. Users can select the toggle at any point in their journey to open and close the ChatBot as needed.
+
+<div class="ws-docs-content-img">
+![2 toggles with open and closed icons.](./img/chatbot-toggle.svg)
+</div>
+
+When there is an unread message from the ChatBot, a notification badge should be placed on the toggle.
+
+<div class="ws-docs-content-img">
+![Toggle with notification badge.](./img/chatbot-toggle-notification.svg)
+</div>
+
+### Sending messages and attaching files
+
+To message the ChatBot, users can type directly into the message bar in the footer or click any included actions. 
+
+<div class="ws-docs-content-img">
+![Elements of a message bar, including input actions.](./img/message-bar-elements.svg)
+</div>
+
+1. **Attach button:** Allows users to upload files from their computer. 
+1. **Use microphone button:** Supports speech recognition to allow users to use voice input.
+1. **Send button:** Allows users to send a typed message. This button should be disabled until a user has input text.
+
+### Using the navigation menu
+
+The ChatBot navigation menu primarily contains a users' conversation history with the ChatBot. Clicking the menu icon opens a side drawer in the ChatBot window. 
+
+<div class="ws-docs-content-img">
+![Conversation history with an options menu opened on a previous conversation.](./img/conversation-history.svg)
+</div>
+
+By clicking into the navigation menu, users can search through previous conversations and perform additional actions, such as sharing a conversation with others. 
+
+## Variations 
```

Comment:
what if we named this Display variations?

Reply:
Our design guidelines template has an h2 variations section (https://www.patternfly.org/ux-writing/writing-patternfly-design-guidelines), so I just added an h3 right below called display variations! lmk if you have thoughts on that approach

Reply:
Gotcha! sounds good